src_c/line.c

```diff
@@ -949,11 +956,26 @@ pg_line_setb(pgLineObject *self, PyObject *value, void *closure)
     return -1;
 }
 
+static PyObject *
+pg_line_getangle(pgLineObject *self, void *closure)
+{
+    double dx = self->line.x2 - self->line.x1;
+    double dy = self->line.y2 - self->line.y1;
```

Comment:
Not that it matters much, but you should move this `dy` calclation after the if statement.